auth-web/src/components/auth/manage-business/BusinessLookup.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <!-- once in Summary state, need to re-mount to reuse this component -->
```

Comment:
The alternative would be to pass in a prop that gets toggled to reset the state, etc. Or a public method that the parent calls.

At the moment, there's no need to reset the state except between instances of the Add Business dialog, so I didn't complicate this.